dandi/tests/test_upload.py

```diff
@@ -35,43 +34,42 @@ def test_upload(local_docker_compose, monkeypatch, tmp_path):
         metadata = yaml.safe_load(fp)
     dandi_id = metadata["identifier"]
 
-    client = girder.get_client(known_instances[dandi_instance].girder)
+    client = girder.get_client(known_instances[dandi_instance_id].girder)
```

Comment:
here and similarly in the other tests:

```suggestion
    client = girder.get_client(local_docker_compose_env["instance"].girder)
```

so we could avoid using `known_instances` as much as possible and take advantage of encapsulation of all relevant information within the fixture.

Reply:
Done.